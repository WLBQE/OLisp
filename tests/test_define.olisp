(define (int a) 1)
(define (double b) 1.2)
(define (bool c) true)
(define (string d) "yes")
(define (string e) "no")
(define (bool f) false)

(print a)
(print b)
(print c)
(print d)
(print e)
(print f)
