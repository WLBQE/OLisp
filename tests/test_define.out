1
1.2
true
yes
no
false
